.npmrc

```diff
@@ -1,2 +1 @@
-shamefully-hoist=true
```

Comment:
I had issues before with this. Do you know what makes this work now?

Reply:
😂 I didn't deep in it,but it work fine now.